src/v2/modules/companies/routes.js

```diff
@@ -0,0 +1,15 @@
+const Joi = require('@hapi/joi');
+const controller = require('./controller');
+
+exports.getCompanies = {
+  method: 'GET',
+  path: '/crm/2.0/companies',
+  handler: controller.getCompanies,
+  options: {
+    validate: {
+      query: {
+        invoiceAccountIds: Joi.string()
```

Comment:
Apparently HAPI understands array params in query strings, e.g.:
```
/some/path?invoiceAccountId[]=x&invoiceAccountId[]=y
```
If this works I wonder if it would be preferable to the above because then the request could then be validated, something like:
```
invoiceAccountId : [
  Joi.string().guid(),
  Joi.array().items(Joi.string().guid())
]
```